src/com/cdd/bao/template/SchemaVocab.java

```diff
@@ -29,8 +29,8 @@ BioAssay Express (BAE)
 	{
 		public String uri;
 		public String label, descr;
-		public String[] altLabels;
-		public String[] externalURLs;
+		public HashSet<String> altLabels = new HashSet<>();
+		public HashSet<String> externalURLs = new HashSet<>();
```

Comment:
We need to think about what `Set` implementation is most suitable. If we want a sorted set, we should use `TreeSet`. If we need to prevent the order of insertion, `LinkedHashSet` is the one.